tests/py/dynamo/conversion/test_as_strided_aten.py

```diff
@@ -0,0 +1,65 @@
+import torch
+from parameterized import parameterized
+from torch.testing._internal.common_utils import run_tests
+from torch_tensorrt import Input
+
+from .harness import DispatchTestCase
+
+
+class TestAsStridedConverter(DispatchTestCase):
+    @parameterized.expand(
+        [
+            (
+                (5, 5),
+                (2, 3),
+                (1, 2),
+                0,
+            ),
+            (
+                (5, 5),
+                (2, 3),
+                (2, 2),
+                1,
+            ),
+            (
+                (20, 20),
+                (2, 3, 2),
+                (2, 2, 2),
+                0,
+            ),
+            (
+                (8, 8, 8),
+                (2, 2, 3),
+                (1, 2, 2),
+                1,
+            ),
+            (
+                (200, 200, 200),
+                (9, 9, 3, 2),
+                (2, 2, 2, 3),
+                1,
+            ),
```

Comment:
Can you add more tests that `size` or `stride` contains 0, and/or the `input` is not a square/cube?

Reply:
Alright, it looks like the output doesn't support empty in torch-trt. I think it's ok to add a validator like you have done, but need to remove the duplicated decorator: https://github.com/pytorch/TensorRT/pull/2735/files#diff-9cbf43cfb62cdf682eef77f6fd5cabc488bb5a91ff09bfee5d593827542b2476R2203